tests/simd/arrays.ml

```diff
@@ -1,6 +1,7 @@
-
 open Stdlib
 
+[@@@ocaml.warning "-37-32-35"]
```

Comment:
* Can we just fix the unused for-loop indexes? That seems relatively less invasive than the other ones.
* I suggest `[@@@ocaml.warning "-unused-constructor" "-unused-value-declaration"]` as it's a bit more self-documenting.

Reply:
Sure, done. Though it seems `[@@@ocaml.warning]` can only take one full name at a time.